config.js

```diff
@@ -18,6 +18,9 @@ let config = {
     email_transport: getTransportConfig(),
     emails_enabled: process.env.EMAILS_ENABLED ? process.env.EMAILS_ENABLED.toLowerCase() === 'true' : true,
     emails_url: process.env.EMAIL_URL ? process.env.EMAIL_URL : 'http://localhost:4010',
+    // temp url
+    temp_email_receiver:"lauwc@nih.gov",
+    temp_test_url: "https://datacommons.cancer.gov/",
```

Comment:
These added variables should be configured as environment variables, given more descriptive names, and also added to the README.md file

I would recommend using SUBMISSION_DOC_URL to replace temp_test_url and should have a default value of an empty string